homework/index.js

```diff
@@ -28,27 +28,74 @@
     });
     return elem;
   }
+  function NewDate(dateString) {
+    const dateTime = new Date(dateString);
+    return dateTime.toLocaleString();
+  }
+
+  function renderRepoDetails(repo, section) {
+    const Mix = createAndAppend('ul', section);
```

Comment:
This variable name should start with a lowercase letter. What does `mix` mean in this context?

Reply:
I think list name will be better than mix